src/parser/sparqlParser/generated/SparqlAutomatic.g4

```diff
@@ -551,8 +551,7 @@ string
     ;
 
 iri
-    : (LANGTAG '@')? (iriref
-    | prefixedName)
+    : (PREFIX_LANGTAG)? (iriref | prefixedName)
```

Comment:
Maybe add a comment here:
"The `PREFIX_LANGTAG` is an extension of the SPARQL standard that allows IRIs like `@en@rdfs:label` which are used in QLever's implementation of language filters.